cli/pcluster/cfnconfig.py

```diff
@@ -44,6 +44,64 @@ def getStackTemplate(region, aws_access_key_id, aws_secret_access_key, stack):
 
 class ParallelClusterConfig(object):
 
+    def __get_efs_parameters(self, __config):
+        # Determine if EFS settings are defined and set section
+        try:
+            self.__efs_settings = __config.get(self.__cluster_section, 'efs_settings')
+            if not self.__efs_settings:
+                print("ERROR: efs_settings defined but not set in [%s] section"
+                                                % self.__cluster_section)
+                sys.exit(1)
+            self.__efs_section = ('efs %s' % self.__efs_settings)
+        except configparser.NoOptionError:
+            pass
+
+        # Dictionary list of all EFS options
+        self.__efs_options = OrderedDict([('shared_dir',('EFSShared_dir',None)), ('efs_fs_id',('EFSFileSystemId','EFSFSId')),
+                                   ('performance_mode',('EFSPerformanceMode','EFSPerfMode')), ('efs_kms_key_id',('EFSKMSKeyId',None)),
+                                   ('provisioned_throughput',('EFSProvisionedThroughput',None)), ('encrypted',('EFSEncryption',None)),
+                                   ('throughput_mode',('EFSThroughput_mode',None))]
+                                  )
+        __validMt = False
+        __throughputmode = None
+        __provisionedthroughput = None
+        try:
+            if self.__efs_section:
+                __temp_efs_options=[]
+                for key in self.__efs_options:
+                    try:
+                        __temp__ = __config.get(self.__efs_section, key)
+                        if not __temp__:
+                            print("ERROR: %s defined but not set in [%s] section" % (key, self.__efs_section))
+                            sys.exit(1)
+                        if key == 'provisioned_throughput':
+                            __provisionedthroughput = __temp__
+                        elif key == 'throughput_mode':
+                            __throughputmode = __temp__
+                        # Separate sanity_check for fs_id, need to pass in fs_id and subnet_id
+                        if self.__sanity_check and self.__efs_options.get(key)[1] == 'EFSFSId':
+                            __validMt = config_sanity.check_resource(self.region, self.aws_access_key_id, self.aws_secret_access_key,
```

Comment:
Please, use underscore separator for python variables, not camelcase. See [PEP8](https://www.python.org/dev/peps/pep-0008/)